src/main/java/com/testmonitor/actions/TestCaseFolders.java

```diff
@@ -123,14 +125,14 @@ public TestCaseFolder create(String name) throws IOException {
     }
```

Comment:
⬆️ The test case folder name you want to create => The name of the test case folder